blocks.js

```diff
@@ -8744,6 +8847,13 @@ TemplateSlotMorph.prototype.evaluate = function () {
     return this.contents();
 };
 
+// TemplateSlotMorph to "scratchblocks"
+
+TemplateSlotMorph.prototype.stringify = function () {
+    var category = this.parent.stringifyCategory() || ' :: grey';
+    return '(' + this.children[0].stringify() + category + ')';
```

Comment:
In Snap, template variables are in doubled rings, like here: `script variables ((a)::grey) ::grey`